helm/flink-operator/crds/flinkdeployments.flink.apache.org-v1.yml

```diff
@@ -9073,6 +9073,13 @@ spec:
                   savepointLocation:
                     type: string
                 type: object
+              jobManagerDeploymentStatus:
+                enum:
```

Comment:
@tweise you will probably need to add a special status for deployment errors. I have not added a state for that as we have no logic in place yet.

Other than that we can probably reuse the reconciliation status error field for the actual error message.